test/server/graphql/v2/collection/ActivitiesCollectionQuery.test.ts

```diff
@@ -114,45 +110,6 @@ describe('server/graphql/v2/collection/ActivitiesCollection', () => {
     expect(result.data.activities.nodes).to.be.sortedBy('createdAt', { descending: true });
   });
 
-  describe('filters activities by attribution', () => {
-    it('Self', async () => {
-      const variables = { account: { legacyId: collective.id }, attribution: 'SELF' };
-      const result = await graphqlQueryV2(activitiesCollectionQuery, variables, admin);
-      result.errors && console.error(result.errors);
-      expect(result.errors).to.not.exist;
-      expect(result.data.activities.totalCount).to.eq(1);
-      expect(result.data.activities.nodes[0].account.slug).to.eq(collective.slug);
-      expect(result.data.activities.nodes[0].fromAccount.slug).to.eq(collective.slug);
-    });
-
-    it('Authored', async () => {
-      const variables = { account: { legacyId: collective.id }, attribution: 'AUTHORED' };
-      const result = await graphqlQueryV2(activitiesCollectionQuery, variables, admin);
-      result.errors && console.error(result.errors);
-      expect(result.errors).to.not.exist;
-      expect(result.data.activities.totalCount).to.eq(1);
-      expect(result.data.activities.nodes[0].fromAccount.slug).to.eq(collective.slug);
-    });
-
-    it('Received', async () => {
-      const variables = { account: { legacyId: collective.id }, attribution: 'RECEIVED' };
-      const result = await graphqlQueryV2(activitiesCollectionQuery, variables, admin);
-      result.errors && console.error(result.errors);
-      expect(result.errors).to.not.exist;
-      expect(result.data.activities.totalCount).to.eq(1);
-      expect(result.data.activities.nodes[0].account.slug).to.eq(collective.slug);
-    });
-
-    it('Hosted', async () => {
-      const variables = { account: { legacyId: collective.id }, attribution: 'HOSTED_ACCOUNTS' };
-      const result = await graphqlQueryV2(activitiesCollectionQuery, variables, admin);
-      result.errors && console.error(result.errors);
-      expect(result.errors).to.not.exist;
-      expect(result.data.activities.totalCount).to.eq(1);
-      expect(result.data.activities.nodes[0].host.slug).to.eq(collective.slug);
-    });
-  });
-
```

Comment:
Maybe we can update that to test our new `includeChildrenAccounts` and `includeHostedAccounts` rather than just removing the tests?

Reply:
Done. 👍🏽